packages/sdk/src/browser-wallets/magiceden/index.ts

```diff
@@ -67,13 +72,13 @@ async function signPsbt(
     inputsToSign,
   }: SatsConnectSignPSBTOptions = { network: "mainnet", inputsToSign: [] },
 ): Promise<BrowserWalletSignResponse> {
-  if (!isInstalled()) {
+  if (!isInstalled(getMeProvider)) {
     throw new BrowserWalletNotInstalledError(
       "Magic Eden not installed or set as prioritised wallet.",
```

Comment:
```suggestion
      "Magic Eden not installed.",
```

https://github.com/ordzaar/ordit-sdk/pull/141/files#r1507341367